packages/contract-deposit/scripts/exec.js

```diff
@@ -45,86 +47,95 @@ const main = async () => {
    * Add the default local network configuration to the SDK
    * to allow this script to run on a local node
    */
-  addDefaultLocalNetwork()
+  addNetworkFromFile()
```

Comment:
@TucksonDev How about adding notes or instructions about addNetworkFromFile in the README of projects that include this function in their scripts? It would let users know they need to complete that step.

Reply:
Great point @mahsamoosavi , thanks!

Since the network will be automatically added to all tutorials, I've added a small section in the main README to explain how to add the network information, and how to obtain the info with the new method from the Orbit SDK.

I've also added a few changes:
- updated the READMEs of the tutorials I've updated (I forgot to change them)
- fixed a small typo in the contract-deposit contract
- Renamed `localNetwork` nomenclature to `customNetwork`

Thank you! :D